gto/cli.py

```diff
@@ -489,18 +497,23 @@ def annotate(
         must_exist=must_exist,
         labels=label,
         description=description,
+        auto_commit=auto_commit,
         # update=update,
     )
 
 
 @gto_command(section=CommandGroups.enriching)
-def remove(repo: str = option_repo, name: str = arg_name):
+def remove(
+    repo: str = option_repo,
+    name: str = arg_name,
+    auto_commit: bool = option_auto_commit,
```

Comment:
> what should be the commit message? I was thinking to use the command used, something like `gto.api.annotate(repo=., name="new-type", auto_commit=True)`

To your first question. Sounds ok to me. It can also be something like "Annotated artifact 'new-type' with path='path/to/it', type='type-of-it', ...".

As a small side quest we could also add `--message` arg (like there is for `gto register`), to allow writing custom commit messages.

Reply:
Good, will do that then.

Regarding the `--message` option though I am not 100% convinced: it adds complexity to a CLI which is already not that straightforward, and I wonder if it will ever be needed. If it is really needed in special circumstances, one can still commit manually with a git command.
I'd suggest to leave it out for now, and think of it only if some user request it. Do you agree?

Reply:
Yep, totally agree. Let's keep it out of the scope for now.